frontend/src/components/WorkoutDetail.tsx

```diff
@@ -36,6 +56,10 @@ export default function WorkoutDetail() {
                 <p><em>{workoutDescription}</em></p>
             </div>
             <button onClick={goToEditPage} className={"editButton"}>Edit</button>
+            <WorkoutPhoto savePhoto={savePhoto}/><br/>
+            {photos?.map((photo, index) =>
```

Comment:
returned das undefined, wenn photos falsy ist? In dem Fall würde ich den ganzen Block konditional zeigen. Aber wird schon so passen, wenn du's getestet hast :)

Reply:
WorkoutPhoto ist nur die Komponente, die das Upload Element bereitstellt. Wenn photos? undefined it, dann werden keine Photos angezeigt. Hat ohne Errors in der Konsole funktioniert.